src/app/pages/ArticlePage/fixtureData.ts

```diff
@@ -75,6 +76,9 @@ const articleDataBuilder = (
     },
     tags: things,
     allowAdvertising,
+    stats: {
+      readTime: readTime || undefined,
```

Comment:
If there's no read time passed in (because it's optional), then it will be `undefined`
```suggestion
      readTime,
```